tests/test_marshmallow_loader.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio marshmallow loader tests."""
+
+from __future__ import absolute_import, print_function
+
+# from invenio_pidstore.models import PersistentIdentifier
+# from invenio_records import Record
```

Comment:
I think those were forgotten to be removed :P

Reply:
mah gad, jesus take the wheel :D
fixing, thanks